api/src/main/java/org/apache/iceberg/PartitionSpec.java

```diff
@@ -19,13 +19,13 @@
 
 package org.apache.iceberg;
 
-import com.google.common.base.Preconditions;
-import com.google.common.collect.ImmutableList;
-import com.google.common.collect.ListMultimap;
-import com.google.common.collect.Lists;
-import com.google.common.collect.Maps;
-import com.google.common.collect.Multimaps;
-import com.google.common.collect.Sets;
+import org.apache.iceberg.shaded.com.google.common.base.Preconditions;
```

Comment:
It may be fewer changes if we chose a different location to shade the dependencies, like `com.google.common.shaded`?

Reply:
Sure, if you'd prefer that, the above would just be an import order change which should be trivial and I was hoping I could use the Spotless plugin to automate the above change but running `spotlessApply` from Gradle doesn't seem to work. There seems to  be some interaction between it and the baseline plugin so I can't see any of the spotless tasks available but I'm sure I'm just missing something?